package.json

```
@@ -10,26 +10,16 @@
     "node": ">=18.0.0"
   },
   "scripts": {
-    "prepare": "husky",
     "publish-modules": "npx lerna run build && npx lerna publish",
     "build": "npx lerna run build",
-    "lint:js": "eslint . --ext .js,.jsx,.ts,.tsx --fix && prettier --write \"packages/**/*.{js,jsx,ts,tsx,json}\" && yarn constraints --fix",
+    "lint:js": "eslint . --ext .js,.jsx,.ts,.tsx && prettier --write \"packages/**/*.{js,jsx,ts,tsx,json}\" && yarn constraints",
```

I think the write flag might have the same effect as fix here:

```suggestion
    "lint:js": "eslint . --ext .js,.jsx,.ts,.tsx && prettier \"packages/**/*.{js,jsx,ts,tsx,json}\" && yarn constraints",
```